### PR TITLE
Circle CI

### PR DESCRIPTION
**Pull request by pat-s**

### CODE REVIEW

File: tic.R [hunk 1]
```diff
@@ -1,5 +1,4 @@
 do_package_checks()
 
-if (ci_on_travis()) {
-  do_pkgdown()
-}
+do_pkgdown()
```
Comment (krlmlr) [inline tic.R]: Are we now building pkgdown on all platforms?

**Review by krlmlr — APPROVED** [review]
Can we get a CircleCI badge in the README?

File: .travis.yml [hunk 2]
```diff
@@ -7,7 +7,7 @@
 before_install:
   - R -q -e 'if (!requireNamespace("remotes")) install.packages("remotes")'
   - R -q -e 'if (getRversion() < "3.2" && !requireNamespace("curl")) install.packages("curl")'
-  - R -q -e 'remotes::install_github("ropenscilabs/tic", upgrade = "always"); print(tic::dsl_load()); tic::prepare_all_stages()'
+  - R -q -e 'remotes::install_github("ropenscilabs/tic@renv", upgrade = "always"); print(tic::dsl_load()); tic::prepare_all_stages()'
```
Comment (krlmlr) [inline .travis.yml]: Do we need this?

**Comment by pat-s** [comment]
I cleaned up and also removed the "deployment" stage from circle -> deployment only on Travis for this repo.

The skeleton including deployment will live in the template in `tic.R`.

**Pull request merged**